content/code_snippets/displaying_images.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Displaying Images
+author: Adam Gerhant
+---
+
+This page will cover how to display an image with ImGui and canvas. The difference between the two methods is that canvas provides better drawing tools, while ImGui provides widgets which can be used interactively. It is possible to use both ImGui and canvas together for a combination of custom shapes and interactive controls.
+
```

Comment:
```suggestion
In your .h, declare your image
`private std::shared_ptr<ImageWrapper> myImage;`

In your onLoad, load your image

{{< highlight cpp "linenos=table" >}}
myImage = std::make_shared<ImageWrapper>(gameWrapper->GetDataFolder() / "myimage.png", true, true);
{{< / highlight >}}
```

Maybe something like a first steps here. Say to load in onLoad?

Reply:
yeah